src/renderer/pages/preference.vue

```diff
@@ -1,11 +1,10 @@
 <template>
   <div class="pref-container">
-    <title-bar v-if="titleBarStyle === 'custom'"></title-bar>
-    <div v-if="titleBarStyle !== 'custom'" class="title-bar"></div>
+    <title-bar v-if="titleBarStyle === 'custom' && !isOsx"></title-bar>
     <side-bar></side-bar>
     <div
       class="pref-content"
-      :class="{ 'frameless': titleBarStyle === 'custom' }"
+      :class="{ 'frameless': titleBarStyle === 'custom' || isOsx }"
```

Comment:
I think you don't need `or isOSX` because the `frameless` class is only used to create a top margin so that the scrollbar is not scrolled in the custom titlebar control buttons.

Reply:
Mark Text in macOs is always a frameless window, so need the top margin.